src/python/WMCore/WMSpec/StdSpecs/Express.py

```diff
@@ -383,7 +385,7 @@ def addAlcaHarvestTask(self, parentTask, parentOutputModuleName,
 
         harvestTaskConditionHelper = harvestTaskCondition.getTypeHelper()
         harvestTaskConditionHelper.setRunNumber(self.runNumber)
-        harvestTaskConditionHelper.setConditionOutputLabel(condOutLabel)
+        harvestTaskConditionHelper.setConditionOutputLabel(condOutLabel+"ALCAPROMPT")
```

Comment:
same here

Reply:
Again, this is a conditional code path that is only called for ALCAPROMPT.